src/SignalR/perf/benchmarkapps/Crankier/Client.cs

```diff
@@ -33,7 +34,8 @@ public async Task CreateAndStartConnectionAsync(string url, HttpTransportType tr
                 }
                 else
                 {
-                    Trace.WriteLine($"Connection terminated with error: {ex.GetType()}: {ex.Message}");
+                    ErrorMessage = $"Connection terminated with error: {ex.GetType()}: {ex.Message}";
+                    Trace.WriteLine(ErrorMessage);
```

Comment:
What do you think about adding a constructor to Client that takes an IAgent and process id so the clients can log directly?

Reply:
Happy to do this, will make the change this evening.